server/config.js

```diff
@@ -129,6 +129,8 @@ module.exports = {
     },
     endpoints: {
       bookings: `${prisonApiBaseUrl}/api/bookings`,
+      offenders: `${prisonApiBaseUrl}/api/offenders`,
+      base: `${prisonApiBaseUrl}/api`,
```

Comment:
I'm a bit confused about this config entry. It's composed from `prisonApiBaseUrl` but is called `base`. Which is the base?

Exposing `prisonApiBaseUrl` is fine here 👍, but we need consistency 🙂. This _could_ be an indicator that `prisonApiBaseUrl` should contain `/api` if all the examples of it append that. Otherwise it's probably safer to just type `/api` where you need it.

Reply:
For refactored code we just use the API base path in order to be more explicit with which endpoint we're calling in the repository